test/integration/forms/action.js

```diff
@@ -603,6 +603,7 @@ context('Patient Action Form', function() {
       })
       .routeActionActivity()
       .routePatientByAction(fx => {
+        fx.data.id = '1';
```

Comment:
this was using an id out of the fixtures.. patients are also loaded along with the actions list, so if you got unlucky and the same fixture was pulled, it would get overridden